PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -38,6 +38,7 @@ using namespace o2::framework;
 using namespace o2::framework::expressions;
 using namespace std;
 
+double massEl = TDatabasePDG::Instance()->GetParticle(11)->Mass();
```

Comment:
- Use named PDG codes.
- Do not use `TDatabasePDG` for masses of particles with named PDG codes.

@victor-gonzalez Please discourage this bad practice systematically.

Reply:
@vkucera I didn’t fully understand this. Should I directly assign the particle’s mass since I’m using it to calculate the particle’s rapidity?

Reply:
Check for instance 
```
constexpr double MassElectron = 0.000510999;
```
in https://github.com/AliceO2Group/AliceO2/blob/dev/Common/Constants/include/CommonConstants/PhysicsConstants.h

Reply:
@TGahlaut1 Please use the `MassElectron` constant (from the linked header) directly without assigning it to a new variable.